src/components/footers/Footer2.tsx

```diff
@@ -0,0 +1,20 @@
+import Image from 'next/image'
+import Link from 'next/link'
+import React from 'react'
+
+export default function Footer2() {
+  return (
+    <div className='fixed flex justify-center items-center bottom-0 w-full border-t-2 border-[#A135E8]' style={{height: '70px'}}>
+        <div className='flex justify-between' style={{width: '84.7%', height: '28px'}}>
+            <h2 className='text-white'
+            style={{width: '290px'}}
```

Comment:
It is possible to use custom sizes in tailwind, it is not necessary to use the 'style' tag